src/main/kotlin/org/vaccineimpact/api/models/CoverageRow.kt

```diff
@@ -18,7 +18,7 @@ data class ScenarioAndCoverageSets(
 )
 
 data class ScenarioTouchstoneAndCoverageSets(
-        val touchstone: Touchstone,
+        val touchstone: TouchstoneVersion,
```

Comment:
I think we should rename all the variable names to reflect the types, otherwise it could continue to be confusing!

Reply:
It makes the change bigger, but you're probably right.